docs/passing-raml-data.md

```diff
@@ -14,11 +14,11 @@ parser.loadApi(apiFileUrl);
 </script>
 ```
 
-Running this code on page load will throw a `TypeError` with the message: `parser.loadApi is not a function`.
+Running this code as the page loads throws a `TypeError` with the message: `parser.loadApi is not a function`.
 
-It's because at the time of execution of this script block the browser knows nothing about the `raml-js-parser` element. At the time it is an instance of `HTMLUnknownElement`.
+At the time of execution of this script, the browser knows nothing about the `raml-js-parser` element. At this time, the element is an instance of `HTMLUnknownElement`. The browser is blocked.
```

Comment:
I'm not sure what ` The browser is blocked.` means in this context. The browser isn't blocked. Script just fails.